### PR TITLE
Backend setup

### PR DESCRIPTION
**Pull request by lazuardyk**
How to run

1. ```go build``` or ```go mod download``` (install dependencies)
2. Create .env (see .env.example)
3. ```go run .```

### CODE REVIEW

**Comment by sonadztux** [comment]
Router sama db migrator & seeder-nya kayaknya perlu dipisah file sendiri lagi deh

**Review by sonadztux — APPROVED** [review]

File: backend/models/visitor.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package models
+
+import (
+	"time"
+
+	"gorm.io/gorm"
+)
+
+type Visitor struct {
+	CookieID    string `gorm:"primary_key"`
```
Comment (asyafalni) [inline backend/models/visitor.go:10]: CookieID ini bukannya harusnya SessionID ya @lazuardyk ?

File: backend/models/visitor_activity.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package models
+
+import (
+	"time"
+)
+
+type VisitorActivity struct {
+	ID        string `gorm:"primary_key"`
+	CookieID  string
```
Comment (asyafalni) [inline backend/models/visitor_activity.go:9]: Ini juga sama https://github.com/nodefluxio/weber/pull/1/files#r722892142

File: backend/models/visitor.go (hunk 1 above)
Comment (lazuardyk) [reply backend/models/visitor.go:10]: oiya bener

File: backend/models/visitor_activity.go (hunk 2 above)
Comment (lazuardyk) [reply backend/models/visitor_activity.go]: Done https://github.com/nodefluxio/weber/pull/1/commits/92a365c8be329ffac5a08c14de81148f1387c923

File: backend/models/visitor.go (hunk 1 above)
Comment (lazuardyk) [reply backend/models/visitor.go]: Done https://github.com/nodefluxio/weber/pull/1/commits/92a365c8be329ffac5a08c14de81148f1387c923

**Review by asyafalni — APPROVED** [review]

**Pull request merged**